### PR TITLE
[Refactor] LanguageScreenViewController Programmatic UI

### PR DESCRIPTION
**Pull request by PranjalDev**
- [x] Implemented LanguageScreenViewController using code
- [x] Tested the feature parity with old implementation
- [x] Integrated the new notification based add word changes (Though I am not much in favour of using notification centre for this purpose and would interested for refactoring this to a protocol/closure based communication)

This is for open Issue [#21 ](https://github.com/misteu/VocabularyTraining/issues/21)

### CODE REVIEW

**Comment by misteu** [comment]
Hey thank you for your PR!

Regarding your last point mentioned: I agree and I'm also not a big fan of using NotificationCenter as I wrote here. I thought it's fine for now to have it for this single purpose. I added my thoughts on that in the PR that added this today: https://github.com/misteu/VocabularyTraining/pull/23#discussion_r1005699027

However, feel free to open an issue for this as well if you want to start working on your idea of a protocol/closure based communication :)

File: VocabularyTrainer/Language Screens/LanguageScreenViewController.swift [hunk 1]
```diff
@@ -11,18 +11,153 @@ import MessageUI
...
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.titleLabel?.font = .systemFont(ofSize: 25)
+        button.setTitle(NSLocalizedString("New Word", comment: "New Word"), for: .normal)
+        button.backgroundColor = BackgroundColor.mediumSpringBud
+        button.layer.cornerRadius = 5.0
+        button.setTitleColor(BackgroundColor.japaneseIndigo, for: .normal)
+        return button
+    }()
+    
+    lazy var hintLabel: UILabel = {
```
Comment (misteu) [inline VocabularyTrainer/Language Screens/LanguageScreenViewController.swift:67]: Can you check the constraints you did set for this?

It is not centered anymore:
grafik

Other than that, great contribution! Thanks so much! :)

Comment (PranjalDev) [reply VocabularyTrainer/Language Screens/LanguageScreenViewController.swift:67]: Have fixed the alignment, Let me know if anything else is a blocker for the merge

**Comment by misteu** [comment]
Thank you for your work! Looks good!

Feel free to pick up any other task you like. I saw your comment regarding the GitHub actions, I'll create an issue for that soon.

**Comment by PranjalDev** [comment]
Hey @misteu can you add necessary hacktoberfest-accepted tag?

**Comment by misteu** [comment]
sure added that!

But as far as I understood, it will be counted anyway as the whole repo has the `hacktoberfest` topic set.

**Pull request merged**